app/views/LocationPreferencesView/ApiKeyModal.tsx

```
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, StyleSheet, Modal, TextInput, TouchableOpacity } from 'react-native';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)